httpevents/request.go

```diff
@@ -42,37 +42,38 @@ func makeRequest(req *http.Request, laddr string) request {
 }
 
 func (r *request) log(logger *events.Logger, depth int) {
+	var mem [10]interface{}
+	var arg = append(mem[:0], r.laddr, r.raddr, r.host, r.method)
+
 	var buf [128]byte
-	var fmt = append(buf[:0], "%{local_address}s->%{remote_address}s - %{host}s - %{method}s %{path}s"...)
+	var fmt = append(buf[:0], "%{local_address}s->%{remote_address}s - %{host}s - %{method}s"...)
```

Comment:
Because for some HTTP methods (CONNECT) there is no path, so it ended up printing two consecutive spaces. Now it only puts the path if it's not empty.